common-arcana.lic

```diff
@@ -365,41 +365,81 @@ module DRCA
     end
   end
 
-  def update_astral_data(data)
+  def update_astral_data(data, settings)
```

Comment:
❌ Is this a breaking change? What will happen to people using `combat-trainer` when this is merged?

Reply:
Yeah, this would need merging a couple days before the CT implementation to avoid issues with most people.

Reply:
I think we need a default value for `settings` or else people will start getting errors, right?